### ISSUE TITLE
ReadMarker backward compatible API /latestreadmarker

### ISSUE THREAD

**Issue by ssardana96** [0chain/0chain#898]
Once #857 is done we should add an API for readLatestMarker to be read from `readmarkers` table
dependent on #895

**Comment by krishnemd** [comment]
@ssardana08 we also need similar for the write_marker. If its already there then please inform it here otherwise please raise a new issue.

**Comment by ssardana96** [comment]
@chauchausoup what's the writemarker API you are looking for, can you share in detail. And where will it be used?

**Comment by krishnemd** [comment]
@ssardana08 so just like the /latestreadmarker,  we need a /latestwritemarker to know the value in explorer app "_Last time an allocation was written/read from._" .

**Comment by ssardana96** [comment]
got it @chauchausoup 
currently we don't have an API `/latestwritemarker`
I'll raise a new issue

**Comment by Sriep** [comment]
It should be the first ReadMarker returned when ordered by decreasing row timestamp.
